### PR TITLE
Cleanup of cassandra loader dev script and swagger version identifier

### PR DESCRIPTION
**Pull request by devshorts**

### CODE REVIEW

File: scripts/build-local-cassandra-dm.sh [hunk 1]
```diff
@@ -1,15 +1,24 @@
...
+MAVEN_URL="https://repo1.maven.org/maven2/io/paradoxical/cassandra.loader/${VERSION}/${JARNAME}"
+RUNNER_LOCATION="$DIR/$JARNAME"
+
+if [ ! -f $RUNNER_LOCATION ]; then
+    wget -O $RUNNER_LOCATION $MAVEN_URL;
+fi
+
+java -jar ${RUNNER_LOCATION} \
+           -f db/scripts \
+           -ip `docker-machine ip $DOCKER_MACHINE_NAME` \
```
Comment (jakeswenson) [inline scripts/build-local-cassandra-dm.sh]: shouldn't this be `--ip`

File: scripts/build-local-cassandra-dm.sh [hunk 2]
```diff
@@ -1,15 +1,24 @@
...
+           -k cassieq \
+           -p 9042 \
+           -createKeyspace -recreateDatabase
```
Comment (jakeswenson) [inline scripts/build-local-cassandra-dm.sh]: Let's double check these options... 
https://github.com/paradoxical-io/cassieq/wiki/Setup%20Database%20Command

File: scripts/build-local-cassandra-dm.sh [hunk 3]
```diff
@@ -1,15 +1,24 @@
...
-
-else
-	echo "Error, did you not dot source this file?"
-	echo "Example:"
-	echo "    . $0"
-fi
\ No newline at end of file
+if [ "$DOCKER_MACHINE_NAME" = "" ]; then
+    log-warn "warning your DOCKER_MACHINE_NAME environment variable was not set. have you run: echo \"\$(docker-machine env <MACHINE-NAME)\"?"
+    DOCKER_MACHINE_NAME=dev
```
Comment (jakeswenson) [inline scripts/build-local-cassandra-dm.sh]: change this to default

**Comment by jakeswenson** [comment]
:heavy_check_mark: lgtm

File: scripts/build-local-cassandra-dm.sh (hunk 3 above)
Comment (devshorts) [inline scripts/build-local-cassandra-dm.sh]: Will do

**Pull request merged**